### PR TITLE
Fix typo in javadoc

### PR DESCRIPTION
**Pull request by nkjackzhang**
Fix typo in javadoc and delete extra white space line.

I have signed and agree to the terms of the Spring Individual Contributor
License Agreement.

### CODE REVIEW

File: spring-core/src/main/java/org/springframework/util/StopWatch.java [hunk 1]
```diff
@@ -289,7 +283,6 @@ public String toString() {
 		return sb.toString();
 	}
 
-
```
Comment (snicoll) [inline spring-core/src/main/java/org/springframework/util/StopWatch.java]: That one should actually stay to separate inner classes from methods.

**Comment by snicoll** [comment]
Thanks for the PR but the current code looks fine as it is. Changing the second `{@code}` would link to the method itself. I think the intention was to use code and have a separate `@see` below.

**Comment by nkjackzhang** [comment]
But in javadoc, there exist extra "#", may be we can just delete the extra characters.

**Comment by snicoll** [comment]
Ah good point, that's a glitch introduced in a6a6aed17f3ef9e9f774f25c14c7e6e5bb5dd2f3 - Fixed in https://github.com/spring-projects/spring-framework/commit/c425e774a3fc96f205832541f630dd205373b940

**Comment by nkjackzhang** [comment]
Thanks!

**Pull request closed without merging**